src/plasma-cash/ethereum-client.ts

```diff
@@ -206,19 +221,25 @@ export class EthereumPlasmaClient {
     const { slot, exitTx, exitBlockNum, prevTx, prevBlockNum, from, gas, gasPrice } = params
     const prevTxBytes = prevTx ? prevTx.rlpEncode() : '0x'
     const exitTxBytes = exitTx.rlpEncode()
-    const bond = this._web3.utils.toWei('0.1', 'ether')
+    // @ts-ignore
+    const bond = ethers.utils.parseEther('0.1')._hex
+
+    let prevBlk
+    if (prevBlockNum !== undefined) {
+      prevBlk = '0x' + prevBlockNum.toString(16)
+    } else {
+      prevBlk = 0
+    }
 
     return this._plasmaContract.startExit(
-      [
-        slot.toString(),
-        prevTxBytes,
-        exitTxBytes,
-        prevTx ? prevTx.proof : '0x',
-        exitTx.proof,
-        exitTx.sig,
-        [prevBlockNum ? prevBlockNum.toString() : 0, exitBlockNum.toString()]
-      ],
-      bond
+      '0x' + slot.toString(16),
```

Comment:
Not loving this `'0x' + slot.toString(16)` everywhere, does `slot.toString()` not work anymore?

Reply:
Yeah unfortunately it doesn't, this is an under the hood difference between web3/ethers